addons/web/static/src/search/search_bar_menu/search_bar_menu.js

```diff
@@ -205,7 +202,16 @@ export class SearchBarMenu extends Component {
      * @param {number} itemId
      */
     openConfirmationDialog(itemId) {
-        const { userId } = this.favoriteItems.find((item) => item.id === itemId);
+        const { userId, removable } = this.favoriteItems.find((item) => item.id === itemId);
+        if (!removable) {
```

Comment:
In your demo, something was wrong with filter deletion.

Reply:
I still have to investigate